3-razor-pages/README.md

```diff
@@ -0,0 +1,222 @@
+# Build a pizza website with Razor Pages
+
+Welcome back! In the last lesson, you got a crash course in C# language fundamentals. Now, you'll get to put those skills to use and build a website using ASP.NET Core and Razor Pages!
+
+## What's ASP.NET Core?
+
+.NET is pretty magical, because you can build all kinds of apps with it – mobile apps, games, desktop apps, web apps, and more. ASP.NET Core is a .NEt toolkit for building web sites, apps, and services with .NET.

```

Comment:
```suggestion
.NET is pretty magical, because you can build all kinds of apps with it – mobile apps, games, desktop apps, web apps, and more. ASP.NET Core is a .NET toolkit for building web sites, apps, and services with .NET.

```